### ISSUE TITLE
Performance + color picker: Colors of the element should change instantly

### ISSUE THREAD

**Issue by miina** [GoogleForCreators/web-stories-wp#4463]
## Feature Description

Currently, when changing the colors in the color picker, the colors of the element don't change right away but only once the user has stopped making changes in the color picker (e.g. stopping after dragging).

The colors should change instantly, ideally, the design input panel would update instantly as well but that's a nice-to-have compared to the element itself.

Originally reported by Paul:
> "...slugginess of the color picker, specifically the fact that when I drag around, the colors on screen don't update instantaneously. In this case, performance isn't just nice-to-have, it actually actively hinders the design process."

From Morten:
> "I guess we can figure out a way to use the transform object to update colors while in flux during a  color drag and only update through the store when you stop dragging (just like we do with moveable)."

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by miina** [comment]
@barklund @bmattb FYI -- working on this and assigned to Sprint 41, however, I won't be able to finish it within this sprint.

**Comment by csossi** [comment]
Verified in QA

**Comment by samitron7** [comment]
Verified in UAT
